Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -5,6 +5,37 @@ protocol PhoneNumberTextFieldDelegate: AnyObject {
     func phoneNumberIsUpdated(number: String, tag: Int)
 }
 
+class BorderView: UIView {
```

Comment:
Might need its own file in a more generic path. I'm thinking of `UI/CommonUI/View/Component`